actors/builtin/multisig/multisig_state.go

```diff
@@ -28,10 +28,19 @@ type State struct {
 	PendingTxns cid.Cid
 }
 
+func (st *State) SetLocked(startEpoch abi.ChainEpoch, unlockDuration abi.ChainEpoch, lockedAmount abi.TokenAmount) {
+	st.StartEpoch = startEpoch
+	st.UnlockDuration = unlockDuration
+	st.InitialBalance = lockedAmount
+}
+
 func (st *State) AmountLocked(elapsedEpoch abi.ChainEpoch) abi.TokenAmount {
 	if elapsedEpoch >= st.UnlockDuration {
 		return abi.NewTokenAmount(0)
 	}
+	if elapsedEpoch <= 0 {
+		return st.InitialBalance
+	}
```

Comment:
This is a bug in master: if the vesting hasn't started yet, `elapsedEpoch` will be negative and the expression below will calculate a locked balance larger than the total locked amount. The change is backwards compatible because it's impossible to configure a multisig with StartEpoch < current epoch until this change.

Reply:
@whyrusleeping prompted me to critically check the case of a multisig operation in the same epoch as creation, and I realised this is _not_ backwards compatible, because the division truncation (#1131) could cause the expression below to return a value that differs from `InitialBalance`.

I'll change the test here to `if elapsedEpoch < 0`, falling through to the slightly wrong behaviour below during the epoch of creation.